CodeBenders/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.10f1
-m_EditorVersionWithRevision: 2019.4.10f1 (5311b3af6f69)
+m_EditorVersion: 2019.4.11f1
+m_EditorVersionWithRevision: 2019.4.11f1 (2d9804dddde7)
```

Comment:
Should we be using this newer version or stay consistent with the current version?

Reply:
I believe that all versions between 2019.4 are inter-compatible. This is just a patch version and we can all upgrade to the latest iteration for bugfixes (IIRC the latest version now is 2019.4.12f1).